pkg/syncer/admissionhandler/validatesnapshotoperation.go

```diff
@@ -36,6 +36,10 @@ func validateSnapshotOperationSupervisorRequest(ctx context.Context, request adm
 	}
 
 	if request.Operation == admissionv1.Create {
+		if featureGateStorageQuotaM2Enabled {
```

Comment:
Can you add a NOTE above line 39 that this is a temporary change and will be changed back before the release?